base/tekton.dev/tasks/mirror-release.yaml

```diff
@@ -43,14 +46,21 @@ spec:
         # Import the public key for verification
         curl -Lv "$(params.gpg-public-key-url)" | gpg --import
 
+        # Grab all of the previous releases for cincinnati
+        PREVIOUS_RELEASES="$(curl https://amd64.origin.releases.ci.openshift.org/api/v1/releasestreams/accepted | jq -r '.["$(params.release-stream)"] | join(",")')"
+
+        # remove any release name that's is less than 4.16
```

Comment:
we need to parametrize this and make sure upgrades are only allowed from the latest stable -> current stable. i.e, if 4.19 is stable, we only allow updates from 4.18 (just the last couple, not all) and 4.19 to the latest 4.19. this will include all releases which is not correct - you cannot upgrade from 4.17 -> 4.19 directly

Reply:
it has to at least hop to it's next minor release , i.e 4.17 -> 4.18 ->4.19